frontend/src/app/spot/styles/sass/components/breadcrumbs.sass

```diff
@@ -0,0 +1,35 @@
+.spot-breadcrumbs
+  @include spot-caption()
+
+  display: flex
+  align-items: baseline
+
+  &--crumb
+    display: flex
+    align-items: center
+    flex-shrink: 0
+    min-width: 0
+
+    > .spot-icon
+      width: $spot-spacing-1-5
+      height: $spot-spacing-1-5
+
+    &_shrink
+      flex-shrink: 1
+
+    &-action-item
+      flex-shrink: 1
+      white-space: nowrap
+      overflow: hidden
+      text-overflow: ellipsis
+
+      &:not(:first-child)
+        margin-left: $spot-spacing-0-25
+
+
+    &-divider
+      font-size: 0.5rem
+      flex-shrink: 0
+
+    &:not(:first-child)
+      margin-left: $spot-spacing-0-25
```

Comment:
I'm confused about the BEM styles again. What are the naming rules we follow?

Reply:
What is it, you are confused about? Can you pinpoint it?

Reply:
single vs. double underscore. And this website: https://en.bem.info/methodology/naming-convention/

Reply:
it is still a bit confusing, but we use the following rules:

- block is kebab case
- block is separated from element with `--`
- modifier are separated with single `_`
- in variable names we use `-` for a space (separator) and a `_` as a replacement for the decimal dot in numbers